templates/actor-sheet.html

```diff
@@ -310,7 +312,7 @@ <h3>All Items</h3>
                     <img src="{{item.img}}" title="{{item.name}}" width="24" height="24"/>
                     <div class="item-name">{{item.name}}</div>
                 </div>
-                <a class="item-control item-delete" title="{{localize 'BITD.TitleDeleteItem'}}"><i class="fas fa-trash"></i></a>
+                <a class="item-control item-delete" title=(localize "BITD.TitleDeleteItem")><i class="fas fa-trash"></i></a>
```

Comment:
I'm pretty sure this should be in double brackets: https://foundryvtt.com/article/localization/